### PR TITLE
Move Settings Functionality into Standalone Class

### PR DESCRIPTION
**Pull request by dallendalton**
This PR serves as the first step in cleaning up and modernizing the TaxJar WooCommerce integration. It primarily consists of moving most of the settings functionality into its own class, and untying the connect class from the integration class. Along side these changes, the classes that were worked on have also been changed to comply with our code standards.

**Click-Test Versions**

- [X] Woo 5.1
- [X] Woo 4.6

**Specs Passing**

- [X] Woo 5.1
- [X] Woo 4.6

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   },
   "require-dev": {
     "squizlabs/php_codesniffer": "*",
-    "wp-coding-standards/wpcs": "0.10.0",
+    "wp-coding-standards/wpcs": "2.3.0",
```
Comment (dallendalton) [inline composer.json:12]: Update wp-coding-standards to ensure we are complying with the latest code standards.

File: includes/class-wc-taxjar-connection.php [hunk 2]
```diff
@@ -2,86 +2,212 @@
 /**
  * TaxJar Connection
  *
```
Comment (dallendalton) [inline includes/class-wc-taxjar-connection.php:4]: Most of the changes to this class were made to make the code easier to follow. The $integration parameter was also removed from the constructor to untie the two classes.

File: includes/class-wc-taxjar-integration.php [hunk 3]
```diff
@@ -112,397 +103,14 @@ public function __construct() {
...
+         *
+		 * @return string
 		 */
-		public function save() {
-			$settings = $this->get_settings();
-			WC_Admin_Settings::save_fields( $settings );
-		}
+		public static function get_regions_uri() {
+		    return self::$app_uri . 'account#states';
+        }
```
Comment (fastdivision) [inline includes/class-wc-taxjar-integration.php:113]: @dallendalton Do we use the `get_regions_uri` method anywhere? Or is this no longer needed?

**Comment by fastdivision** [comment]
Reviewed all of the code, looking good so far 👍 Going to review again later before approving.

**Review by poops — APPROVED** [review]

**Review by fastdivision — APPROVED** [review]

**Pull request merged**